<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetMoney</class>
 <widget class="QWidget" name="GetMoney">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1704</width>
    <height>1121</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>HP Simplified</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelGetMoneyText">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>40</y>
     <width>631</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>46</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Rahat tulossa...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelGetMoneyImage">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>190</y>
     <width>1011</width>
     <height>591</height>
    </rect>
   </property>
   <property name="text">
    <string>Image</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupReceiptScreenOrPaper">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>920</y>
     <width>600</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="buttonReceiptScreen">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Näytölle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonReceiptPaper">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Paperille</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAskReceiptScreenOrPaper">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tulostetaanko kuitti näytölle vai paperille?</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupAskReceipt">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>920</y>
     <width>600</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="buttonReceiptYes">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Kyllä</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonReceiptNo">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ei</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAskReceipt">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>571</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tulostetaanko kuitti?</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
